crypto/include/nss_fips.h

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024, Red Hat, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *   Redistributions of source code must retain the above copyright
+ *   notice, this list of conditions and the following disclaimer.
+ *
+ *   Redistributions in binary form must reproduce the above
```

Comment:
This is not a public header in this project so will not be (and should not be) installed in binary installations, how to ensure this condition is meet?

Reply:
I got permission from the original author to remove the restrictive part of the header and make it public. See https://bugzilla.mozilla.org/show_bug.cgi?id=1922760#c3